.github/labeler.yml

```diff
@@ -0,0 +1,10 @@
+dashboard:
```

Comment:
you might want to add the labeler action to our github workflow so it can read this file and label the PR.

Reply:
omg, you're right! I somehow missed adding that file 🙈